superset-frontend/src/explore/components/controls/DateFilterControl/DateFilterLabel.tsx

```diff
@@ -314,7 +314,7 @@ export default function DateFilterLabel(props: DateFilterControlProps) {
       <Divider />
       <div>
         <div className="section-title">{t('Actual time range')}</div>
-        {validTimeRange && <div>{evalResponse}</div>}
+        {validTimeRange && <div>{t(evalResponse)}</div>}
```

Comment:
Again, you can't translate a variable (please see e.g. https://stackoverflow.com/questions/34579316/flask-babel-how-to-translate-variables for an explanation why this is the case)

Reply:
chanhe has been move in this PR : https://github.com/apache/superset/pull/20080
We can discuss about translation of varaible in the ohter PR
Without the t() here, there are any translation, do you have an idea, where I can translate that